internal/policygen/iam.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policygen
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/terraform"
+	"github.com/hashicorp/terraform/states"
+)
+
+type binding struct {
+	Type   string
+	ID     string
+	Role   string
+	Member string
+}
+
+func generateIAMBindingsPolicies(resources []*states.Resource, outputDir string) error {
```

Comment:
Is the purpose of this function to eventually do more than just call `bindings()`? (I assume yes, from the TODOs, otherwise it should be removed)

Reply:
Yea. This function will also generate the output policy files. Given this PR is already pretty large, I put them as TODOs and will do in following PRs.